sdk/core/Azure.Core/src/ResourceIdentifier.cs

```diff
@@ -278,7 +278,7 @@ private static ResourceType ChooseResourceType(ReadOnlySpan<char> resourceTypeNa
             return "Invalid resource id.";
         }
 
-        private static ReadOnlySpan<char> PopNextWord(ref ReadOnlySpan<char> remaining)
+        private static ReadOnlySpan<char> PopNextWord(scoped ref ReadOnlySpan<char> remaining)
```

Comment:
I must have missed the announcement on this. Having read about it now, seems a bit like rust lifetime parameters...but simple.